features/step_definitions/collections.js

```diff
@@ -84,3 +85,48 @@ When(/I check if collection "(.*?)" exists on index "(.*?)"$/, function (collect
   return stepUtils.getReturn.call(this, 'collectionExists', index, collection, cb);
 });
 
+When('I create a collection {string}:{string}', function (index, collection) {
+  return this.api.createCollection(index, collection);
+});
+
+Then('The mapping dynamic field of {string}:{string} is {string}', function (index, collection, dynamicValue) {
+  return this.api.getCollectionMapping(index, collection)
+    .then(({ result }) => {
+      const expectedValue = dynamicValue === 'the default value' ? this.kuzzleConfig.services.db.dynamic.toString() : dynamicValue;
+
+      should(result[index].mappings[collection].dynamic)
+        .not.be.undefined()
+        .be.eql(expectedValue);
+    });
+});
+
+When('I update the mapping of {string}:{string} with {string}', function (index, collection, rawMapping) {
+  const mapping = eval(`() => (${rawMapping})`)(); // eslint-disable-line no-eval
+
+  return this.api.updateMapping(index, collection, mapping);
+});
+
+Then('The mapping properties field of {string}:{string} is {string}', function (index, collection, rawMapping) {
+  const includeKuzzleInfo = rawMapping === 'the default value' ? true : false;
```

Comment:
This line basically means: `const includeKuzzleInfo = true ? true : false;`

```suggestion
  const includeKuzzleInfo = rawMapping === 'the default value';
```

Reply:
Ahah indeed :sweat_smile:

Reply:
Ahah indeed :sweat_smile: